istioctl/pkg/install/k8sversion/version_test.go

```diff
@@ -76,21 +76,15 @@ func TestExtractKubernetesVersion(t *testing.T) {
 		errMsg   error
 		isValid  bool
 	}{
-		{
-			version:  version1_16,
-			expected: 16,
-			errMsg:   nil,
-			isValid:  true,
-		},
 		{
 			version:  version1_17,
 			expected: 17,
 			errMsg:   nil,
 			isValid:  true,
 		},
 		{
-			version:  version1_8,
-			expected: 8,
```

Comment:
I'm more concerned with this change to 1.18 from 1.8 as we are no longer checking a 1.x (where x is a single digit).
I would entertain added an `.18 is that is needed.

Reply:
Added back `1.8` test case. Thanks.